lib/config/types.ts

```diff
@@ -193,7 +193,8 @@ export interface RegexManagerTemplates {
   extractVersionTemplate?: string;
   registryUrlTemplate?: string;
 }
-export interface RegExManager extends RegexManagerTemplates {
+export interface CustomManager extends RegexManagerTemplates {
+  customType: string;
```

Comment:
```suggestion
  customType?: 'regex';
```

maybe more strict?

Reply:
In future we want more custom managers, should I create a unoon type which can be used here?

Reply:
yes please